### PR TITLE
Bump version to 0.11.0

### PR DESCRIPTION
**Pull request by syntapy**

### CODE REVIEW

**Comment by frostming** [comment]
Hi, I just checked the change and found `is_tomlkit` isn't necessary. So I created #195 to remove it, would need another look if you are fine with it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -274,6 +280,7 @@
 - Fixed raw strings escaping.
 
 [unreleased]: https://github.com/sdispater/tomlkit/compare/0.10.2...master
```
Comment (frostming) [inline CHANGELOG.md:282]:
```suggestion
[unreleased]: https://github.com/sdispater/tomlkit/compare/0.11.0...master
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.11.0] - 2022-05-25
```
Comment (frostming) [inline CHANGELOG.md:5]:
```suggestion
## [0.11.0] - 2022-05-24
```

**Review by frostming — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
...
+
+## Fixed
```
Comment (frostming) [inline CHANGELOG.md:7]:
```suggestion
## Added
```

Comment (syntapy) [reply CHANGELOG.md:7]: Oh yeah thats right. Added not fixed

**Review by frostming — APPROVED** [review]

**Pull request merged**